tests/test_fxa_test_account.py

```diff
@@ -0,0 +1,46 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+import re
+
+import pytest
+from fxa.errors import ClientError
+
+from fxapom.fxapom import DEV_URL, FxATestAccount, PROD_URL
+
+
+@pytest.mark.skip_selenium
+class TestFxATestAccount(object):
+
+    def check_login(self, acct):
+        # attempt to login, if unsuccessful will throw a ClientError
+        try:
+            acct.login()
+        except ClientError:
+            raise AssertionError('Login unsuccessful. Account not properly created/verified.')
```

Comment:
This will conceal the ClientError, which might make investigation difficult if there are failures. As the unexpected exception will cause the test to error, can we just leave it?